### PR TITLE
Prepare release 1.3.0

### PR DESCRIPTION
**Pull request by romanstrobl**
Merge of all changes for release `1.3.0` from `develop` into `master`.

### CODE REVIEW

**Review by petrdvorak — APPROVED** [review]
Looks good

File: docs/Releases.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ In order to consolidate the information about the current versions, we have intr
     - `Z` is the bugfix version. Upgraded in case a hotfix is needed in some `X.Y.*` version.
 
 ## List of Releases
-
+- [PowerAuth 2022.04](./PowerAuth-2022.04.md)
```
Comment (banterCZ) [inline docs/Releases.md:14]: Formatting: I would keep an empty line after heading.

File: docs/Readme.md [hunk 2]
```diff
@@ -21,6 +21,10 @@ A typical use-case for PowerAuth protocol would be assuring the security of a mo
     - [Activation Upgrade](./Activation-Upgrade.md)
 - [List of Used Keys](./List-of-used-keys.md)
 
+## Tutorials
```
Comment (banterCZ) [inline docs/Readme.md:24]: dtto

**Review by banterCZ — APPROVED** [review]

File: docs/Releases.md (hunk 1 above)
Comment (romanstrobl) [reply docs/Releases.md:14]: I will not fix this as part of pull request, this pull request is from `develop` into `master`. Feel free to raise a separate issue for this for the next release.

File: docs/Readme.md (hunk 2 above)
Comment (romanstrobl) [reply docs/Readme.md:24]: dtto

**Pull request merged**